pymine/types/nbt.py

```diff
@@ -345,8 +345,7 @@ def __init__(self, name: str, data: list) -> None:
     def pack_data(self) -> bytes:
         if len(self) > 0:
             return BufferUtil.pack("b", self[0].id) + BufferUtil.pack("i", len(self)) + b"".join([t.pack_data() for t in self])
-        else:
-            return BufferUtil.pack("b", 0) + BufferUtil.pack("i", 0)
+        return BufferUtil.pack("b", 0) + BufferUtil.pack("i", 0)
```

Comment:
Again, there should be a newline following the change in indentation from the if statement above this line.